mods/d2k/sequences/infantry.yaml

```diff
@@ -196,10 +196,15 @@ medic: # actually thumper
 		Facings: -8
 		Transpose: true
 		Tick: 120
-	heal: DATA.R8
+	thump: DATA.R8
 		Start: 1458
 		Length: 5
 		Tick: 480
```

Comment:
Something isn't right here. I think you are missing `Transpose: true` because on your branch the deployed thumper is still holding his (other?) thumping device:

![image](https://cloud.githubusercontent.com/assets/756669/10749607/99713338-7c6c-11e5-8500-749b2774b862.png)

However I believe you want it to look like

![image](https://cloud.githubusercontent.com/assets/756669/10749584/5a990c26-7c6c-11e5-8353-1adec3df2df2.png)

compare https://github.com/OpenRA/OpenRA/pull/9019/.